src/Theme/PageTemplate.elm

```diff
@@ -0,0 +1,28 @@
+module Theme.PageTemplate exposing (..)
+
+import Css exposing (Style, batch)
+import Html.Styled as Html exposing (div, text)
+import Html.Styled.Attributes exposing (css)
+
+
+type alias Msg =
+    Never
+
+type alias PageUsingTemplate msg =
+    { content : Maybe (Html.Html msg) }
+
+view : PageUsingTemplate msg -> Html.Html msg
+view pageInfo =
+    div [][ 
```

Comment:
I think we'll want more semantic markup here.

Maybe ask @aaaaargZombies for a template, but do we want this to be `main` and maybe also add a header / footer?

Reply:
ohhhh * penny dropping noise *

Reply:
Ok I've made those changes, so we're now
- body
    - header
    - main
    - footer

I've left the content as views in the main.elm to avoid clashing with anyone else's pattern for this for now.